api/v1alpha1/nonadminbackup_types.go

```diff
@@ -21,16 +21,24 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// NonAdminBackupPhase is a simple one high-level summary of the lifecycle of an NonAdminBackup.
+// +kubebuilder:validation:Enum=New;BackingOff;Created
+type NonAdminBackupPhase string
+
+const (
+	// NonAdminBackupPhaseNew - NonAdminBackup resource was accepted by the OpenShift cluster, but it has not yet been processed by the NonAdminController
+	NonAdminBackupPhaseNew NonAdminBackupPhase = "New"
+	// NonAdminBackupPhaseBackingOff - Velero Backup object was not created due to NonAdminBackup error (configuration or similar)
+	NonAdminBackupPhaseBackingOff NonAdminBackupPhase = "BackingOff"
+	// NonAdminBackupPhaseCreated - Velero Backup was created. The Phase will not have additional informations about the Backup.
+	NonAdminBackupPhaseCreated NonAdminBackupPhase = "Created"
+)
+
 // NonAdminBackupSpec defines the desired state of NonAdminBackup
 type NonAdminBackupSpec struct {
-	// https://github.com/vmware-tanzu/velero/blob/main/pkg/apis/velero/v1/backup_types.go
-
 	// BackupSpec defines the specification for a Velero backup.
```

Comment:
I think I saw in the code a check if this was not nil

adding `// +kubebuilder:validation:Required` would be valid here :question: